packages/mco/utils/disaster-recovery.ts

```diff
@@ -0,0 +1,12 @@
+import { ODF_MINIMUM_VERSION  } from '../constants/dr-policy';
+
+// CreateUniqueName function creates a name from managed cluster names
+export const createUniqueName = (names: string[]) => {
+    names.sort(function (a, b) {
+        return a.localeCompare(b);
+    });
```

Comment:
```suggestion
    names.sort()
```
default sorting is ascending.